gradio/components.py

```diff
@@ -1173,7 +1178,7 @@ def postprocess(self, y):
         Returns:
         (str): base64 url data
         """
-        if self.type == "auto":
+        if self.output_type == "auto":
```

Comment:
If we remove `self.output_type`, we can just rewrite this to be:

          if isinstance(y, np.ndarray):
              dtype = "numpy"
          elif isinstance(y, PIL.Image.Image):
              ...

Reply:
Gave a reply at upper comment